generators/app/index.js

```diff
@@ -179,21 +185,25 @@ module.exports = generators.Base.extend({
     },
 
     askForGithubAccount: function () {
-      var done = this.async();
-
-      githubUsername(this.props.authorEmail, function (err, username) {
-        if (err) {
-          username = username || '';
-        }
-        this.prompt({
-          name: 'githubAccount',
-          message: 'GitHub username or organization',
-          default: username
-        }, function (prompt) {
-          this.props.githubAccount = prompt.githubAccount;
-          done();
+      if (this.options.githubAccount) {
```

Comment:
I rearranged this to prevent the system from making the server call. This also removes the need for `this.async`